File: atividade-4/001_max3/001_max3.hs
module Main where

-- Defina a função max3 tal que max3 x y z é o máximo entre x, y e z. 

max3 :: Int -> Int -> Int -> Int
max3 x y z
   | x >= y && x >= z = x
   | x < y && y > z = y
   | otherwise = z

main = do
  a <- readLn :: IO Int
  b <- readLn :: IO Int
  c <- readLn :: IO Int
  print $ max3 a b c

File: atividade-3/modulo-4/001_max3/001_max3.js
//Defina a função max3 tal que max3 x y z é o máximo entre x, y e z.

const { x, y, z } = { x: 1, y: 2, z: 3 };

const max3 = (x, y, z) => {
  if (x >= y && x >= z) return x;
  if (x < y && y > z) return y;

  return z;
};

console.log(max3(x, y, z));
